main/reference/zoe-api/zcfseat.md

```diff
@@ -1,9 +1,11 @@
 # ZCFSeat Object
 
 Zoe uses **seats** to access or manipulate offers. Seats represent active
-offers and let contracts and users interact with them. Zoe has two kinds
-of seats. **ZCFSeats** are used within contracts and with **zcf** methods.
-**UserSeats** represent offers external to Zoe and the contract.
+offers and let contracts and users interact with them. Two kinds of seats
+represent a single position, and allow interaction from within a contract
+and from the outside. **ZCFSeats** are used within contracts and with **zcf** methods.
+**UserSeats** represent offers external to Zoe and the contract. The two
```

Comment:
```suggestion
**UserSeats** represent offers external to the contract. The two
```
Specifying "external to Zoe" doesn't add any useful information for the readers, correct?

Reply:
yep